compiler/ballerina-parser/src/test/resources/expressions/xml-template/xml_template_assert_24.json

```diff
@@ -306,8 +306,21 @@
                                                       "isMissing": true
                                                     },
                                                     {
-                                                      "kind": "STRING_LITERAL",
-                                                      "value": "b${v2}c\\}cc{d{}e}{f{\u0027 bar2\u003d\u0027aaa{${v1}}b\\${b"
+                                                      "kind": "BINARY_EXPRESSION",
+                                                      "children": [
+                                                        {
+                                                          "kind": "STRING_LITERAL",
+                                                          "value": "b${v2}c\\}cc{d{}e}{f{\u0027 bar2\u003d\u0027aaa{${v1}}b\\${b"
+                                                        },
+                                                        {
+                                                          "kind": "PLUS_TOKEN",
+                                                          "isMissing": true
+                                                        },
+                                                        {
+                                                          "kind": "IDENTIFIER_TOKEN",
+                                                          "value": "b"
+                                                        }
+                                                      ]
```

Comment:
Why is this change required?

Reply:
adding this expression is causing a change in this recovery. So I changed the assert